service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -41,62 +38,48 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
         Object migrationId = caseDetails.getData().get(MIGRATION_ID_KEY);
 
-
-        if ("FPLA-2491".equals(migrationId)) {
-            run2491(caseDetails);
-        }
-
         if ("FPLA-2379".equals(migrationId)) {
             run2379(caseDetails);
         }
-
-        if ("FPLA-2501".equals(migrationId)) {
-            run2501(caseDetails);
+        if ("FPLA-2525".equals(migrationId)) {
+            run2525(caseDetails);
         }
 
         caseDetails.getData().remove(MIGRATION_ID_KEY);
         return respond(caseDetails);
     }
 
-    private void run2491(CaseDetails caseDetails) {
+    private void run2525(CaseDetails caseDetails) {
         CaseData caseData = getCaseData(caseDetails);
 
-        if ("SN20C50018".equals(caseData.getFamilyManCaseNumber())) {
+        if ("SN20C50028".equals(caseData.getFamilyManCaseNumber())) {
             List<Element<HearingBooking>> hearings = caseData.getHearingDetails();
 
-            LocalDate expectedHearing1Date = LocalDate.of(2021, Month.FEBRUARY, 5);
-            LocalDate expectedHearing2Date = LocalDate.of(2021, Month.MARCH, 4);
-
-            if (hearings.size() < 5) {
-                throw new IllegalArgumentException(
-                    format("Case has %s hearing(s), expected at least %s", hearings.size(), 5));
+            if (hearings.isEmpty()) {
+                throw new IllegalArgumentException("No hearings in a case");
             }
 
-            Element<HearingBooking> hearing1 = hearings.get(3);
-            Element<HearingBooking> hearing2 = hearings.get(4);
+            Element<HearingBooking> hearingToBeRemoved = hearings.get(0);
```

Comment:
are we sure it is first hearing in the list? I am guessing yes

Reply:
yes, this is the only one.